source/moto_payments/index.html.md.erb

```diff
@@ -16,43 +16,66 @@ This means you can take payments:
 
-## If your PSP is Worldpay or ePDQ
+Our product roadmap is based on feedback. [Contact us](/support_contact_and_more_information/#contact-us) if you'd like us to investigate taking MOTO payments through payment links.
 
-If your PSP is Worldpay or ePDQ, you must first:
+## Set up MOTO payments
 
-- [create a new GOV.UK Pay service](https://selfservice.payments.service.gov.uk/create-service/register) for MOTO payments
-- contact your PSP and ask for a new merchant code for MOTO payments
+You can set up MOTO payments on either a test account or a live account.
 
-[Contact Government Banking](https://www.gov.uk/government/groups/government-banking-service-gbs) instead if you use them.
+MOTO payment screens are the same as online payment screens, excluding the following:
 
-## Set up your account
+- MOTO payments screens do not have a section for billing address
+- [Strong Customer Authentication](/set_up_3dsecure/#set-up-strong-customer-authentication-3ds1-and-3ds2) does not apply to MOTO payments
 
-Email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:govuk-pay-support@digital.cabinet-office.gov.uk) and we'll:
+## Set up MOTO payments on a test account
 
-- review your [Payment Card Industry Data Security Standards (PCI DSS) assessment](/security/#payment-card-industry-pci-compliance) to check you can take payments over the phone and by post
-- enable MOTO payments in your service
+[Contact GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk) to ask us to enable MOTO payments on your test account.
 
-## Create a MOTO payment
+You do not need any provide us with any supporting information.
+
+## Set up MOTO payments on a live account
+
+How you set up MOTO payments on a live account differs depending on what payment service provider (PSP) you use.
+
+### Set up MOTO payments on a live account with Worldpay or ePDQ
+
+If you use Worldpay or ePDQ, you must:
+
+- use a service for MOTO payments that is separate to your online payments service
+- process MOTO payments on a specific MOTO merchant code
+
+1. Create a separate MOTO payments service if you do not have one already by signing in to your [GOV.UK Pay account](https://selfservice.payments.service.gov.uk/) and going to __My services__. Select __Add a new service__ to add a new service for MOTO payments to your account.
+
+1. If you do not already have a MOTO merchant code, ask for a new code by contacting your PSP or Government Banking as required. You will use this merchant code when you [connect your new service to your PSP](/switching_to_live/#go-live).
+
+1. Send your [PCI DSS Self Assessment Questionnaire](/security/#payment-card-industry-pci-compliance) and service name to [GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk).
 
-Make sure you've collected your user's:
+We will review your PCI DSS compliance. Your PCI DSS compliance requirements are different for MOTO payments compared to online payments.
```

Comment:
Might want to explain why we need it. Elsewhere we say that Pay looks after a service's PCI compliance, whereas for MOTO we don't. This was a problem for one department and other organisations have queried.

I'm happy to help on writing that, @jonathanglassman.

Reply:
+1, I think we should explain the how the responsibility model differs for MOTO payments up front (ie in the opening para's of this section)

Reply:
@sfount @miriamalice @heathd I agree that it is important howeber I think that sort of information should live in https://www.payments.service.gov.uk/take-payments-by-phone-or-post/ and we link to it from here, as it is supporting information for the technical task

Reply:
@stevenjmesser will slack you

Reply:
I've raised https://github.com/alphagov/pay-product-page/issues/288